crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -70,17 +70,17 @@ async fn mapper_publishes_init_messages_on_startup() {
         &mut mqtt,
         [
             (
-                "te/device/main///twin/c8y_Agent",
+                "c8y/inventory/managedObjects/update/test-device",
                 json!({
-                    "name": "thin-edge.io",
-                    "url": "https://thin-edge.io",
-                    "version": version
+                    "c8y_Agent" : {
+                        "name": "thin-edge.io",
+                        "url": "https://thin-edge.io",
+                        "version": version
+                    }
```

Comment:
I am not asking to fix, but I found it interesting. In the [c8y docs](https://cumulocity.com/docs/device-integration/fragment-library/#agent-information), `c8y_Agent.version` is marked mandatory. Though I know we can put anything via REST API or JSON over MQTT.